enterprise/templates/enterprise/grant_data_sharing_permissions.html

```diff
@@ -27,113 +27,7 @@
-                    <span class="fa-stack fa-lg dropdown-icon-container" aria-hidden="true">
-                      <span id="consent-policy-dropdown-icon-surround" class="fa fa-circle fa-stack-2x" aria-hidden="true"></span>
-                      <span id="consent-policy-dropdown-icon" class="fa fa-chevron-right fa-stack-1x" aria-hidden="true"></span>
-                    </span>
-                  </button>
-
-                  <div class="consent-policy" id="consent-policy" style="display:none;">
-                    <p>{{ sharable_items_header }}</p>
-                    <ul class="consent-policy-bulletpoints">
-                      {% for item in sharable_items %}
-                        <li>{{ item }}</li>
-                      {% endfor %}
-                    </ul>
-                    <p>{{ sharable_items_footer }}</p>
-
-                    <p>{{ sharable_items_note_header }}</p>
-                    <ul class="consent-policy-bulletpoints">
-                      {% for item in sharable_items_notes %}
-                        <li>{{ item }}</li>
-                      {% endfor %}
-                    </ul>
-
-                    <p>
-                      <a class="pull-right" href="#">{{ policy_return_link_text }}</a>
-                    </p>
-                  </div>
-                </div>
-                <div id="consent-confirmation-modal" class="modal" role="dialog" aria-modal="true" aria-labelledby="modal-header-text" aria-hidden="true" tabindex="-1">
-                  <div class="modal-dialog modal-md" role="document">
-                    <div id="consent-confirmation-modal-content" class="modal-content">
-                      <div class="modal-header">
-                        <button id="modal-close-button" type="button" class="close" data-dismiss="modal" aria-hidden="true" aria-label="{% trans "Close" %}">
-                          {% fa_icon 'times' %}
-                        </button>
-                      </div>
-                      <div class="modal-body">
-                        <header class="modal-header-wrapper">
-                          <h2 id="modal-header-text" class="modal-header-text">
-                            {{ confirmation_modal_header }}
-                          </h2>
-                        </header>
-                        <p>{{ confirmation_alert_prompt|safe }}</p>
-                        <p>{{ confirmation_alert_prompt_warning }}</p>
-                        <button class="consent-agreement-button" id="modal-no-consent-button">{{ confirmation_modal_affirm_decline_text }}</button>
-                        <button class="failure-link" id="review-policy-link">{{ confirmation_modal_abort_decline_text }}</button>
-                      </div>
-                    </div>
-                  </div>
-                </div>
-              </main>
+              {% include "enterprise/grant_data_sharing_permissions_agreement_content.html" %}
```

Comment:
Can you name the partial `_data_sharing_policy.html`? The leading `_` is a convention to indicate that a template is included in another template.

Reply:
yes, I'll make that change